virt/install/uninstalling-virt.adoc

```diff
@@ -7,6 +7,11 @@ toc::[]
 You can uninstall {VirtProductName} by using the {product-title}
 xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console].
 
+[NOTE]
+====
+If you use the command-line interface (CLI) to uninstall {VirProductName} and delete the Operator's Subscription, you must also remove the CSV for the Operator by using the CLI.
+====
+
```

Comment:
I have two comments:
1. I think the note about the uninstallation via the CLI should be located at the bottom of the section, not in the prerequisites.
2. To uninstall CNV / Openshift virtualization from the cluster, the user must delete the HyperConverged Cluster object prior to the subscription & CSV removal, otherwise tier-2 operators (the ones which weren't deployed by OLM) would remain running on `openshift-cnv` namespace.

Reply:
@orenc1
Is there a command set that you can share to be included in the write-up?
Thanks, lavanya

Reply:
> @orenc1
> Is there a command set that you can share to be included in the write-up?
> Thanks, lavanya

Deleting the HyperConverged Custom Resource:
`oc delete HyperConverged kubevirt-hyperconverged -n openshift-cnv`

Deleting the subscription:
`oc delete subscription kubevirt-hyperconverged -n openshift-cnv`

Deleting the CSV:
`CSV_NAME=$(oc get csv -n openshift-cnv -o=custom-columns=:metadata.name)`
`oc delete csv ${CSV_NAME} -n openshift-cnv`

Reply:
@orenc1 - I have added the virt uninstallation procedure - please review. 
Questions:
--In the command set, I have inserted virt instead of cnv. Since our product name is changing, I am assuming that the command set in the 2.4 product that we ship will show the new command set with virt instead of cnv? Is this correct?
--Please review the prerequisites section and advise if you need any changes to be made.
Thanks, Lavanya

Reply:
Hi @lmandavi ,
- The prerequisites section looks fine to me (though I would change the order - first you need the `oc` client installed on the workstation, and then point it to the cluster)
- Although the product name has been changed from _Container-native Virtualization_ to _Openshift virtualization_, the namespace in which the product is installed remains "_openshift-cnv_", to support a manageable upgrade path. This was approved by the product marketing director:
https://mojo.redhat.com/docs/DOC-1216821

> Q: Do we have to change the names of rpms, channels, subscriptions, workspace names, etc., that currently use "cnv"?
> 
> A: The PM and engineering teams have agreed that this name change does NOT require the changing of these types of elements, only public-facing text, documentation, web pages, UIs, etc.

Reply:
Hi @orenc1, Reviewed your input. Will get back to you with updated content. Thanks for the link to the mojo page and the info about continuation as openshift-cnv. Will connect back with you soon with updated content.

Reply:
Finalized the content with your input.